backend/chainlit/config.py

```diff
@@ -54,6 +54,9 @@
 # Enable third parties caching (e.g LangChain cache)
 cache = false
 
+# Set langchain cache path (default is .langchain.db inside config dir)
+# lc_cache_path = './.my-cache-path.db'
```

Comment:
If this is a default config option, it seems better to uncomment it, in line with other default config options.